manifests/init.pp

```diff
@@ -46,6 +46,10 @@
 #
 # $repo_export_dir::    Directory to create for repository exports
 #
+# $pulp_db_username::   Username for mongo
+#
+# $pulp_db_password::   Password for mongo
```

Comment:
We aren't doing this kind of exposure for the Candlepin database, maybe we should. Also, if you change this to just database we won't have to update it when Pulp switches off mongo.